File: AreaTriangulo.java
import java.util.Scanner;
class AreaTriangulo{
static Scanner teclado=new Scanner(System.in);
public static void main(String [] arg){
  //Definir variables y otros
    System.out.println("Hola Mundo");
    int b, h, area;
    //Datos de entrada
    System.out.println("Ingrese la base:");
    b=teclado.nextInt();
    System.out.println("Ingrese la base:");
    h=teclado.nextInt();
    //Proceso
    area=(b*h)/2;
    //Datos de Salida 
    System.out.println("Area de un Triangulo:"+area);
}
}

File: AreaTriangulo.cpp
#include <iostream>
int main() {
    // Definir Variables y otros
    printf("Ejercicio 01: Area de un triangunlo\n");
    int b=0, h=0, area=0;
    //Datos de entrada 
    printf("Ingrese Base:");
    scanf("%i", &b);
    printf("Ingrese Altura:");
    scanf("%i", &h);
    //Proceso
    area=(b*h)/2;
    //Datos de Salida
    printf("Ingrese Base: %i \n", area);
    return 0;
}